src/Speckle.Sdk/Common/HashCode.cs

```diff
@@ -7,21 +7,24 @@ namespace Speckle.Sdk.Common;
 /// </summary>
 public readonly struct HashCode : IEquatable<HashCode>
 {
-  private const int EmptyCollectionPrimeNumber = 19;
-  private readonly int value;
+  private const int EMPTY_COLLECTION_PRIME_NUMBER = 19;
+  private readonly int _value;
 
   /// <summary>
   /// Initializes a new instance of the <see cref="HashCode"/> struct.
   /// </summary>
   /// <param name="value">The value.</param>
-  private HashCode(int value) => this.value = value;
+  private HashCode(int value)
```

Comment:
why drop the short constructors?  just a style thing

Reply:
I may not have intended to change this...
I don't really mind which syntax is used

Reply:
reverted constructor, and made [ide0021](https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/ide0021) a suggestion